csv_location_test.go

```diff
@@ -0,0 +1,35 @@
+//go:build unit
+
```

Comment:
More of a comment than a need to change. Tabular tests are really helpful for test cases like this. You can see examples in the [pt-tools](https://github.com/UCLALibrary/pt-tools) and [go-festerize](https://github.com/UCLALibrary/go-festerize) repos. I've mostly only written tabular tests so far in go.

Reply:
Just updating with result of Slack conversation Angela and I had: I deleted the test. It really wasn't useful since it was just testing a basic struct, which we can assume works as advertised.